test/main.js

```diff
@@ -112,6 +112,31 @@ describe('dotenv-expand', function () {
       obj['SOME_ENV'].should.eql('production')
       done()
     })
+
+    it('does not expand inline escaped dollar sign', function (done) {
+      var dotenv = {
+        parsed: {
+          'INLINE_ESCAPED_EXPAND_BCRYPT': '\\$this\\$is\\$bcrypt'
```

Comment:
Should we go ahead and add the test case you brought up in #20? That way it's easy to make sure that any future changes don't break the handling of a real bcrypt string?

Reply:
Will do